pages/common/ned.md

```diff
@@ -0,0 +1,36 @@
+# ned
+
+> Is like `grep` but with powerful replace capabilities.
+> Unlike `sed`, as it isn't restricted to line oriented editing.
+
+- Recursively search starting in the current directory, ignoring case:
+
+`ned -iR '{{^[dl]og}}' {{.}}`
+
+- Search always showing colored output:
+
+`ned --colors '{{^[dl]og}}' {{.}}`
+
+- Search never showing colored output:
+
+`ned --colors=never '{{^[dl]og}}' {{.}}`
+
+- Search ignoring certain files:
+
+`ned -R --exclude '{{*.htm}}' '{{^[dl]og}}' {{.}}`
```

Comment:
Oops! I think I missed one. Is there a long form of `-R`?

Reply:
Dammit, I missed it to! Done.

Reply:
Thanks!